src/content/docs/serverless-function-monitoring/aws-lambda-monitoring/get-started/compatibility-requirements-aws-lambda-monitoring.mdx

```diff
@@ -12,29 +12,18 @@ Before [enabling serverless monitoring](/docs/serverless-function-monitoring/aws
 
 ## Recommended AWS Lambda language runtimes [#languages]
 
-* Node.js: `nodejs14.x`, `nodejs16.x`, `nodejs18.x`
-* Python: `python3.7`, `python3.8`, `python3.9`, `python3.10`, `python3.11`
-* Go: `provided`, `provided.al2`
-* Java: `java8.al2`, `java11`
-* .NET Core: `dotnetcore3.1`
-
-AWS has older runtimes for these languages as well, but AWS has not [chosen to support](https://docs.aws.amazon.com/lambda/latest/dg/runtimes-extensions-api.html)
-the latest Lambda APIs with those older runtimes. Integration for older runtimes requires a different strategy, but
-is possible.
-
-Python and Node.js are by far the most popular languages in the Lambda ecosystem. The
-[New Relic Lambda Layers](https://layers.newrelic-external.com/)
-for Node.js and Python include the very latest New Relic agent version, and provide rich instrumentation with minimal
-configuration, right out of the box.
-
-Similarly, Go uses the New Relic Go agent. We recommend building self-hosting Go lambda functions, because the
-latest AWS Lambda APIs are not supported in the AWS `go1.x` runtime. Fortunately, this is well supported by the AWS
-Lambda SDK for Go. New Relic recommends keeping the agent module up to date. Support is limited for agent versions
-older than 3.9.0.
-
-To minimize performance impact, we've taken a different approach with Java and .NET Core. New Relic provides
-[OpenTracing](https://opentracing.io/) SDKs for these runtimes. This approach requires a bit more code to integrate.
-For newer runtimes, such as .NET 6, [Trace your .NET Lambda functions with New Relic and OpenTelemetry](/docs/serverless-function-monitoring/aws-lambda-monitoring/opentelemetry/lambda-opentelemetry-dotnet/) is available.
+* Node.js: `nodejs16.x`, `nodejs18.x`, `nodejs20.x`
+* Python: `python3.7`, `python3.8`, `python3.9`, `python3.10`, `python3.11`, `python3.12`
+* Go: `provided.al2`
+* Java: `java8.al2`, `java11`, `java17`
```

Comment:
Ideally, we'd remove the Java runtime as well since, like .NET, it is based on the deprecated OpenTracing project.